index.html

```diff
@@ -5,11 +5,12 @@
 
 <div class="hero">
   <div class="hero__logo">
-    <img alt="RAC logo" src="/img/homepage_logo.png" />
+    <img alt="" src="https://assets.rockarch.org/assets/img/hero_logo.png" />
   </div>
   <h1 class="hero__text">
-    Explore our library collection. <br /> Find books to aid your archival research. <br /> Identify relevant secondary sources.
+    The Rockefeller Archive Center is a major repository and research center for
```

Comment:
You want to keep the library-specific text here.

Reply:
Good catch! I missed that.